Documentation/community/roadmap.rst

```diff
@@ -0,0 +1,218 @@
+documentation, blog posts, example configurations, presentations, training
+courses, testing and more. Check the :ref:`dev_guide` documentation to understand how to get
+involved with code contributions, and the `Get Involved`_ guide for guidance on
+contributing blog posts, training and other resources. 
+
+CNCF Graduation
+~~~~~~~~~~~~~~~
+
+Given the widespread adoption of Cilium, `CNCF Graduation`_ is a natural goal for
+the project. We believe that the majority of the criteria are already met, but
+the following still need to be addressed: 
+
+* `CII Best Practice Badge`_
+* Third-party security audit
+
+.. _rm-cilium-service-mesh:
+
+Cilium Service Mesh 
+~~~~~~~~~~~~~~~~~~~
+
+Our eBPF-accelerated Service Mesh is the main focus for
+major enhancement, and it's a natural evolution of Cilium's networking
+capabilities. We released a beta at the end of 2021 and had very valuable
+feedback from our user community. The next steps we'd like to take for Cilium
+Service Mesh (in no particular order) are: 
+
+* Integration of service mesh branch into mainline Cilium
+* Graduating Ingress to stable
+  
+  * 100% upstream conformant
+  * Visibility via Hubble
+  * Support for additional annotations
+ 
+* Graduating Prometheus metrics and OpenTelemetry collector to stable
+* Using Kubernetes as servicemesh control plane 
+ 
+  * Simple to use sidecar-free service mesh configured using Kubernetes Services
+    and Ingress with support for additional annotations
+
+* Graduating EnvoyConfig CRD to stable
+* Extended sample Grafana dashboards for L7 visibility
+* SMI integration 
+* SPIFFE integration
+* Gateway API Integration
+* Next-generation mutual authentication datapath framework
+
+  * Support for integrated runtime identity
+  * SSL-based mutual authentication
+  * Support for any network protocol
+
+* Performance benchmarking
+
+.. _rm-clustermesh:
+
+ClusterMesh
+~~~~~~~~~~~
+
+Core :ref:`ClusterMesh<clustermesh>` is stable and widely adopted. Future extensions include: 
+
+* Service affinity
+* Cluster health checks
+* :ref:`External Workloads<external_workloads>` graduating to stable
+
+
+.. _rm-advanced-networking:
+
+Advanced Networking features
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are several advanced networking features currently in beta, several of
+these are already in production use with a set of adopters. We expect the
+following features to graduate to stable:
+
+.. 
+   Docs comment: Wondering why the Maglev link looks different from the rest? For some 
+   reason, possibly to do with the heading level that Maglev Consistent Hashing has 
+   in the docs, defining a target for it just didn't want to work. Please feel free to 
+   make this better! 
+
+* :ref:`gsg_encryption` (IPsec & WireGuard)
+* :ref:`BGP<bgp>`
+* :ref:`bandwidth-manager`
+* :ref:`Local Redirect Policy<local-redirect-policy>`
+* :ref:`Egress Gateway<egress-gateway>`
+* :ref:`CiliumEndpointSlice<gsg_ces>`
+* `Maglev Consistent Hashing`_
```

Comment:
bandwidth manager and egress GW will be stable in 1.12, so they could probably go to your major feature status? (Maglev is actually used in prod, too. We should move it out of beta from the doc.)

Reply:
This will (I hope) be live before 1.12 is released, so I'll do an update after that. But are we saying that Maglev is stable in 1.11?

Reply:
Maglev-related PR: https://github.com/cilium/cilium/pull/19541, hopefully stable in 1.12